R/settings.R

```diff
@@ -94,54 +94,59 @@ read_config_file <- function(config_file) {
     return(NULL)
   }
 
+  # clickable link for eventual error messages.
+  config_link <- cli::style_hyperlink( # nolint: object_usage_linter. TODO(#2252).
+    cli::col_blue("config"),
+    paste0("file://", config_file)
+  )
   config <- new.env()
   if (endsWith(config_file, ".R")) {
     load_config <- function(file) sys.source(file, config, keep.source = FALSE, keep.parse.data = FALSE)
     malformed <- function(e) {
-      cli_abort(c(
-        "Malformed config file, ensure it is valid R syntax.",
-        conditionMessage(e)
-      ))
+      cli_abort(
+        "Malformed {config_link} file, ensure it is valid R syntax.",
```

Comment:
This will look like `Malformed <a>config</a> file`, right? WDYT about doing `Malformed config file <a>basename(file)</a>` instead? (ditto elsewhere)?

Reply:
I proposed something. Only show the link in the `malformed error msg to avoid repetition.